content/Day-6/_index.md

```diff
@@ -0,0 +1,94 @@
+
+* Formatting and organizing documentation for clarity
+* Reviewing and editing documentation for accuracy and completeness
+* Knowledge of our audience (readers)
+* Visual aids 
+* Step-by-step instructions and conditional call outs 
+
+## Writers Framework 
+
+Our first steps help us create the framework for our documentations and determine what we are writing about, who we are writing for and what we want our readers to know before we start writing. These are the fundamental parts of our writing. This is a universal framework that we can use to create presentations, research papers, fiction and non-fiction artifacts. 
+
+### Writers Framework 
+1. Who needs to read this document?    
+2. When should they read this document and what should it include/exclude?
+3. What is the document about? 
+4. How can they use this information going forward?  
+5. Identify Key Points or Objectives 
+
+### Situation: Using The Writers Framework For Password Reset 
+
+**A customer password reset** password reset is the most common help desk issue. It’s also the easiest one to resolve. Let’s walk through an example of a framework. We work in a call center and we get multiple calls a day. Assume that we have a note taking application for call notes and a knowledge management system for research. You are a Technical Service Analyst and you receive a call for a password reset. 
+
+**Note:** this is a simple example that may exist in smaller organizations. In a larger organization you will use several knowledge management systems and potentially two or three ticketing systems for reporting issues. For our example, we are keeping things simple. 
+
+### 1. Who needs to read this document
+
+We may have two different people in attendance (our audience). For the sake of this example, let’s call these two audience members Tier 1 Help Desk and Tier 2 Support. We could be writing for both our team members and the escalation team so we should include all relevant screenshots and capture notes for each respective team to review.  Using what we’ve learned from Module 1 Effective Communications, we can gather more information about the issue by asking open ended questions about the reported issue and what the customer is experiencing. Then using active listening to tune into anything that sounds like a larger issue. For example, if the caller/client says I got a 404 message several times when trying to login but it worked after the fifth (5) try when I refreshed the browser. This could mean there’s an issue with the application routing and should be escalated to the network or application team for further analysis. 
```

Comment:
This can potentially link back to module 1 for ease of access / review if needed